markerclustererplus/src/markerclusterer.js

```diff
@@ -361,6 +377,18 @@ ClusterIcon.prototype.getPosFromLatLng_ = function (latlng) {
   return pos;
 };
 
+/**
+ * Sync the eventDiv_ size and position with the display div_
+ */
+ClusterIcon.prototype.syncDivStyle = function (vars) {
+  if (vars) {
+    Object.assign(this.eventDiv_, vars);
```

Comment:
This line is only only used to sync the `title` attribute because it is the only thing besides of CSS updated on `this.div_` outside of the `add` hook.